Dockerfile

```diff
@@ -1,15 +1,7 @@
-FROM node:16.19.0-alpine
+FROM node:18.12.0-alpine
```

Comment:
18.12.1 if possible

Reply:
Meh, I read wrongly your message 😆

Reply:
Fixed in [441ce11](https://github.com/Compensate-Operations/cloudbuild-node/pull/23/commits/441ce11ee18360ea94681408c7b5a9da3ca6fc0b)

NOTE: I don't guarantee that this will fly, as I removed a bunch of requirements, but I want to see first hand from the CI how it blows up and only then add the dependencies that are required.